scripts/test

```diff
@@ -4,7 +4,6 @@
 # Script for executing the test suite
 #
 
-npm run typings
 npm run compile
 
 mocha -- $(find dist -name '*.spec.js')
```

Comment:
hmm ... hmm ... Coverage reports would be pretty bad :) – The main part in this case is to test the reducer and this part is covered.